server/container/guice/configuration/src/main/java/org/apache/james/utils/ConfigurationPerformer.java

```diff
@@ -25,7 +25,7 @@
 
 public interface ConfigurationPerformer {
 
-    void initModule();
+    void initModule() throws Exception;
```

Comment:
I don't like this, I prefer the previous version with a better propagation

Reply:
Well, what's the exact point? We are always stupidly propagating these exceptions...

Does it disallow you to write proper error handling code? Imo not at all. But for 100% of today cases it makes it shorter.

I don't see how it harms, I see some little fluency gain...

Reply:
I'll drop it as it's quite orthogonal to this PR, but I still paintain my claim on this.

Reply:
at least it allows to avoid the ugly sneakyThrow

Reply:
Instead of a sneaky throw in a single class, you get 50+ uggly propagate... Don't seem better to me.